pallets/staking/src/lib.rs

```diff
@@ -2168,44 +2182,23 @@ decl_module! {
             Self::deposit_event(RawEvent::InvalidatedNominators(caller_id, caller, expired_nominators));
         }
 
-        /// Enables individual commissions. This can be set only once. Once individual commission
-        /// rates are enabled, there's no going back.  Only Governance committee is allowed to
-        /// change this value.
-        #[weight = (800_000_000, Operational, Pays::Yes)]
-        pub fn enable_individual_commissions(origin) {
-            T::RequiredCommissionOrigin::ensure_origin(origin.clone())?;
-            let key = ensure_signed(origin)?;
-            let id = <identity::Module<T>>::get_identity(&key);
-
-            // Ensure individual commissions are not already enabled
-            if let Commission::Global(_) = <ValidatorCommission>::get() {
-                <ValidatorCommission>::put(Commission::Individual);
-                Self::deposit_event(RawEvent::IndividualCommissionEnabled(id));
-            } else {
-                return Err(Error::<T>::AlreadyEnabled.into());
-            }
-        }
-
         /// Changes commission rate which applies to all validators. Only Governance
         /// committee is allowed to change this value.
         ///
         /// # Arguments
-        /// * `new_value` the new commission to be used for reward calculations
+        /// * `new_cap` the new commission cap.
         #[weight = (800_000_000, Operational, Pays::Yes)]
-        pub fn set_global_commission(origin, new_value: Perbill) {
+        pub fn set_commission_cap(origin, new_cap: Perbill) {
             T::RequiredCommissionOrigin::ensure_origin(origin.clone())?;
-            let key = ensure_signed(origin)?;
-            let id = <identity::Module<T>>::get_identity(&key);
+            let id = <identity::Module<T>>::get_identity(&ensure_signed(origin)?);
```

Comment:
The origin will be a committee origin. Using `ensure_signed` here can result in unexpected behavior. Consider hardcoding the did to GC did or at least use something like `unwrap_or` to use GC did as a fallback instead of panicking.